### DISCUSSION TITLE
Access to web gui of router

### DISCUSSION THREAD

**Discussion by mvmaksnet** [Ysurac/openmptcprouter#2159] [Q&A]
Hello. How can I have access to router from anywhere?

I can open 192.168.100.1 from local lan, but I want to check router from remote location.

**Comment by EticWeb** [comment]
Hi. I guess you need to add port forwarding and nat rule. Have a look on the wiki documentation.

**Reply by romainbacque** [reply]
Hi. Which wiki documentation are you referring to?

**Reply by Ysurac** [reply]
https://github.com/Ysurac/openmptcprouter/wiki

**Reply by romainbacque** [reply]
I am on it and specifically on https://github.com/Ysurac/openmptcprouter/wiki/Port-forwarding and what I found is not working

**Reply by Ysurac** [reply]
It's working. Check debug section to debug.

**Reply by romainbacque** [reply]
I double checked it all and it is unfortunately not working.

**Reply by julianguttzeit** [reply]
I would be very happy if anybody could specify, how its possible to check in the network remotely. I can't figure out the Port Forwarding Settings good enough, that it's working. For example I want to reach my UnRaid Server in the Office. The UnRaid Server has the option to be accessed remotely. The UnRaid Server takes the IP from the MPTCP Server. How do I open the Port? Do I need to to SERVER Configurations als well? Or only Router Settings...?

But now already a big thank you to the provision of the software! Great piece of work!

**Reply by julianguttzeit** [reply]
> I would be very happy if anybody could specify, how its possible to check in the network remotely. I can't figure out the Port Forwarding Settings good enough, that it's working. For example I want to reach my UnRaid Server in the Office. The UnRaid Server has the option to be accessed remotely. The UnRaid Server takes the IP from the MPTCP Server. How do I open the Port? Do I need to to SERVER Configurations als well? Or only Router Settings...?
> 
> But now already a big thank you to the provision of the software! Great piece of work!

Nevermind... GOT IT! The documentation was a little bit confusing. Source adress and Destination adress are exactly the other way around.
